File: cfgs/pythoncfgs/GOOGLE_CASE_GIVEN_SENTENCE.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;statement;foo_1 = '';9;
3;statement;foo_1 = foo_2 + fun_1();10;
4;statement;foo_1 = 1;11;
5;conditional;foo_1 < foo_2;12;
6;statement;return foo_1;19;
7;conditional;foo_1[foo_2] == ' ' and foo_3 <= foo_4;13;
8;statement;foo_1 = foo_2 + fun_1();17;
9;statement;foo_1 = foo_2 + 1;18;
10;statement;foo_1 = foo_2 + ' ' + fun_1();14;
11;statement;foo_1 = foo_2 + 1;15;
12;exit;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;7;
5;12;
6;8;10;
7;9;
8;5;
9;11;
10;9;
11;

File: cfgs/java_cfgs_foo/GOOGLE_CASE_GIVEN_SENTENCE.java
Nodes
1;special;;;
2;statement;int n = fun_1();14;
3;statement;String s1 = "";15;
4;statement;s1 = foo_1 + fun_1(fun_2(0));16;
5;statement;int i = 1;17;
6;conditional;foo_1 < foo_2;18;
7;conditional;(fun_1(foo_2) == ' ') && (foo_3 < foo_4);20;
8;statement;s1 = (foo_1 + " ") + fun_1(fun_2(foo_3 + 1));21;
9;statement;i++;22;
10;statement;s1 = foo_1 + fun_1(fun_2(foo_3));24;
11;statement;i++;19;
12;statement;return foo_1;26;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;12;
6;8;10;
7;9;
8;11;
9;11;
10;6;
11;13;
12;
